### DISCUSSION TITLE
Fine tuning Mesh Offset for smaller bed clips?

### DISCUSSION THREAD

**Discussion by eecue** [Jyers/Marlin#395] [Q&A] [answered]
I recently switched from the default clips to Swiss Clips which only extend about 3.5mm onto the bed.  What variables in the firmware do I tweak to reflect that before compiling?

**Comment by Jyers** [comment, marked as answer]
`PROBING_MARGIN` and `MESH_INSET` can both be adjusted to allow for the maximum probing area.

**Reply by eecue** [reply]
This worked like a charm thanks!
